### ISSUE TITLE
zadig 2.8 does not work on sony A7iv when updating to Ver 2.01

### ISSUE THREAD

**Issue by ASASS568** [ma1co/Sony-PMCA-RE#457]
![P_20231029_213543](https://github.com/ma1co/Sony-PMCA-RE/assets/149306815/0f35a3f7-fa79-42e6-8a88-e2bee0405944)
zadig 2.8 does not work on sony A7iv when updating to Ver 2.01

**Comment by arunbose1989** [comment]
does your camera still show that it is connected using mass storage? I was not getting the same error, but it kept saying check usb connection(in japanese)anyhow all I did was just click on camera info button under camera info tab, and it just established the connection again.

**Comment by ASASS568** [comment]
I'm still in mass storage mode, maybe because zadig 2.8 is not compatible with ver 2.01

**Comment by arunbose1989** [comment]
Yeah likely a compatibility issue then.

**Comment by ZzMrNUDEzZ** [comment]
Hello, I'm considering buying the Sony A7IV. I'm wondering if the issues discussed above have been resolved.

**Comment by ASASS568** [comment]
I have unlocked it, you can buy the international version

**Comment by ZzMrNUDEzZ** [comment]
Currently, I am living in Japan, so I can only purchase the Japanese domestic version. I want to switch to English for easier interaction with online lessons. Can you guide me on how to unlock the language with firmware ver. 2.01?

**Comment by dreambold** [comment]
@ZzMrNUDEzZ You can try it on Ubuntu or  Mac machine, it's just easy!

**Comment by ZzMrNUDEzZ** [comment]
> @ZzMrNUDEzZ You can try it on Ubuntu or Mac machine, it's just easy!

Thanks you very much.

**Comment by dreambold** [comment]
Just let me know if you need any addtional help!

**Comment by ASASS568** [comment]
https://github.com/mcuee/libusb-win32/releases/download/snapshot_1.2.7.4/zadig_2.8_mod.exe
Use this version, put the device in msc mode, and then just do as usual like this video
https://www.youtube.com/watch?v=BJhheKXs39A
https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbU90UElrU3dmaVJuQ0pMNVFYMFYtZTA4UXhxUXxBQ3Jtc0trcnlzNzlXTzd5QVZLQjViMEpPZV9KTG9PSUtTeTN6MXVHTFJIendSd21ycWM2RC1JMEVEQ19FX212Qk9CUlBmNkRMb1YwX0kwLVRCZV9ENzlhTy05VVJfN0VJUFAxQ0hfaDJvaXNoNkYzY1k1NHBOcw&q=https%3A%2F%2Fapp.box.com%2Fs%2F0u2gyrrtwoeosvghpv902e8cwzk537i3&v=BJhheKXs39A

**Comment by ZzMrNUDEzZ** [comment]
> https://github.com/mcuee/libusb-win32/releases/download/snapshot_1.2.7.4/zadig_2.8_mod.exe Use this version, put the device in msc mode, and then just do as usual like this video https://www.youtube.com/watch?v=BJhheKXs39A https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbU90UElrU3dmaVJuQ0pMNVFYMFYtZTA4UXhxUXxBQ3Jtc0trcnlzNzlXTzd5QVZLQjViMEpPZV9KTG9PSUtTeTN6MXVHTFJIendSd21ycWM2RC1JMEVEQ19FX212Qk9CUlBmNkRMb1YwX0kwLVRCZV9ENzlhTy05VVJfN0VJUFAxQ0hfaDJvaXNoNkYzY1k1NHBOcw&q=https%3A%2F%2Fapp.box.com%2Fs%2F0u2gyrrtwoeosvghpv902e8cwzk537i3&v=BJhheKXs39A

Thanks you very much.

**Comment by DiracSeaSea** [comment]
Thanks a lot, a question, can I apply future firmware updates after lenguage hack?

**Comment by dragoscatalinus** [comment]
> ASASS568

Did you manage to get it working?

**Comment by DiracSeaSea** [comment]
> > ASASS568
> 
> Did you manage to get it working?

Yes, working for 2.01, also heard that updating to 3.0 after modding can erased the mod. But no Sure.

**Comment by badokun** [comment]
I'm about to buy a Japanese a7iv . I believe the latest firmware is v4. Since you can't downgrade firmware do we know what's the latest version this hack will work for? I've seen v5 for another Sony camera fx3 works

**Comment by nicustraton** [comment]
I can solve Sony, Nikon, Canon cameras firmware problems/issues after firmware update etc.

If you want to unlock region for your Sony FX3/FX30 camera or other model, in order to have latest firmware and PAL-NTSC, write me on whatsap +40727005523, country code Romania (+40) and number 727005523 or mail me: nicustraton.baron at gmail dot com.

You just have download a USB remote software, enter my IP, connect the camera with USB cable in Image Transfer (MSC) mode to a windows PC/laptop with Windows OS and Intel CPU and share the connection. I will unlock camera first and you pay after.

PS: if you have an older firmware version on camera and you know you have your camera from grey market If you want to update firmware to latest version, if you will update, you will lose PAL/NTSC so I recommend to let me unlock region first, update firmware to latest version and then unlock region again.

* this is a safest way becauseI will backup your camera original settings and firmware before any procedures.
